From: Antonio Ospite <ospite@studenti.unina.it>
Date: Fri, 6 Jan 2012 14:31:50 +0100
Subject: do_not_use_version_number_in_the_system_path

Global installation expects system patch to be something like
/usr/lib/blender/scripts/ without version numbers.
---
 source/blender/blenlib/intern/path_util.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/source/blender/blenlib/intern/path_util.c b/source/blender/blenlib/intern/path_util.c
index 383e0e8..ebe4e2a 100644
--- a/source/blender/blenlib/intern/path_util.c
+++ b/source/blender/blenlib/intern/path_util.c
@@ -89,7 +89,7 @@
 #  else /* new XDG ~/blender/.config/ */
 #    define BLENDER_USER_FORMAT			"%s/blender/%s"
 #  endif // WITH_XDG_USER_DIRS
-#  define BLENDER_SYSTEM_FORMAT			"%s/blender/%s"
+#  define BLENDER_SYSTEM_FORMAT			"%s/blender"
 #endif
 
 /* local */
@@ -1025,7 +1025,7 @@ static int get_path_system(char *targetpath, const char *folder_name, const char
 
 	system_base_path = (const char *)GHOST_getSystemDir();
 	if (system_base_path) {
-		BLI_snprintf(system_path, FILE_MAX, BLENDER_SYSTEM_FORMAT, system_base_path, blender_version_decimal(ver));
+		BLI_snprintf(system_path, FILE_MAX, BLENDER_SYSTEM_FORMAT, system_base_path);
 	}
 	
 	if(!system_path[0])
-- 
